### ISSUE TITLE
[New Work] Make the chart selector component more eye appealing 

### ISSUE THREAD

**Issue by ayan-biswas0412** [indeplot/indeplot#71]
**What is the work should be done please be clear and short about the work**
It is now a simple drop down menu make it more beatutiful show your creativity

**What the work is all about ? Is there any specific files that need to be changed please include those.**
see the src/component/chatSelector.js file and make changes accordingly

**The Best way to implement this (if any) **
No specific way

**Additional context** 
NA

**Comment by bensbits91** [comment]
I'd like to try this. Can you assign it to me, or should I just go for it? Thanks, Ben

**Comment by ayan-biswas0412** [comment]
@bensbits91 yeah go ahead

**Comment by bensbits91** [comment]
@ayan-biswas0412 [I submitted a PR](https://github.com/indeplot/indeplot/pull/96). Let me know if you'd like me to change anything.

I didn't add any dependencies. I used icons from fortawesome, which is already installed. There are good icons for Bar, Line and Pie. I couldn't find any suitable icons in the project for Polar or Doughnut. The icons are easy to replace if you want to change them. Let me know if you want me to find, add and use icons for Polar or Doughnut.

![image](https://user-images.githubusercontent.com/56796209/96641065-fb1ab700-12d8-11eb-914f-14f05a48fac5.png)


Thanks!
Ben
